intellij/test/junit/saros/intellij/filesystem/IntelliJReferencePointManagerTest.java

```diff
@@ -0,0 +1,107 @@
+package saros.intellij.filesystem;
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.roots.ModuleRootManager;
+import com.intellij.openapi.vfs.VirtualFile;
+import org.easymock.EasyMock;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import saros.filesystem.IPath;
+import saros.filesystem.IReferencePoint;
+
+public class IntelliJReferencePointManagerTest {
+
+  IReferencePoint referencePoint;
+  IPath fileReferencePointRelativePath;
+  IPath folderReferencePointRelativePath;
+  IPath projectFullPath;
+  IntelliJReferencePointManager intelliJReferencePointManager;
+  VirtualFile file;
+  VirtualFile folder;
+  VirtualFile moduleRoot;
+  String moduleRootPath;
+  ModuleRootManager manager;
+
+  @Before
+  public void setup() {
+    // moduleRootPath = "path/to/foo";
+    moduleRootPath = "";
+    referencePoint = EasyMock.createMock(IReferencePoint.class);
+
+    EasyMock.replay(referencePoint);
+
+    intelliJReferencePointManager = new IntelliJReferencePointManager();
+  }
+
+  @Test
+  public void testCreateReferencePoint() {
+    IReferencePoint referencePoint =
+        IntelliJReferencePointManager.create(
+            createModule(createModuleRootManager(createVirtualRoot(moduleRootPath, "/asd", null))));
+    Assert.assertNotNull(referencePoint);
+  }
+
+  @Test
+  public void testModulePutIfAbsent() {
+    Module module =
+        createModule(createModuleRootManager(createVirtualRoot(moduleRootPath, "/asd", null)));
+    IReferencePoint referencePoint = IntelliJReferencePointManager.create(module);
+    intelliJReferencePointManager.putIfAbsent(module);
+    Module module2 = intelliJReferencePointManager.getModule(referencePoint);
+
+    Assert.assertNotNull(module2);
+  }
+
+  @Test
+  public void testPairPutIfAbsent() {
+    Module module =
+        createModule(createModuleRootManager(createVirtualRoot(moduleRootPath, "/asd", null)));
+    intelliJReferencePointManager.putIfAbsent(referencePoint, module);
+    Module module2 = intelliJReferencePointManager.getModule(referencePoint);
+
+    Assert.assertNotNull(module2);
+  }
+
+  @Test
+  public void testGetResource() {
```

Comment:
As codacy already detected: This test has no assert

Reply:
I rewrote the tests